src/widgets/configure-related-items/configure-related-items.ts

```diff
@@ -0,0 +1,16 @@
+import connectConfigureRelatedItems, {
+  ConfigureRelatedItemsConnectorParams,
+} from '../../connectors/configure-related-items/connectConfigureRelatedItems';
+import { WidgetFactory } from '../../types';
+
+type ConfigureRelatedItems = WidgetFactory<
```

Comment:
actually why not import `ConfigureRelatedItemsWidgetFactory` from `../../connectors/configure-related-items/connectConfigureRelatedItems` here directly ?

Reply:
95b754ec794f39bb3e5c4b457476d255e9741bf4